charts/policy-reporter/values.yaml

```diff
@@ -71,12 +71,21 @@ resources: {}
   #   memory: 20Mi
   #   cpu: 5m
 
+networkPolicy:
+  enabled: false
+
 # enable policy-report-ui
 ui:
   enabled: false
+  # Set this to true when enabling the UI and networkPolicy.enabled is true.
+  networkPolicy:
```

Comment:
Could you please add this new variable also in the values yaml of the ui subchart? This would work but it be helpful to have all available subchart configuration also in the subchart values.yaml

Reply:
The same for the kyvernoPlugin subchart

Reply:
Forgive my ignorance but this suggestion seems to be at odds with DRY? I'm not shooting down the suggestion, just explaining why I didn't do that originally and am interested in knowing your thoughts.

Reply:
To be consistent with all other configurations of the subcharts the networkPolicy.enabled configuration should also be part of the subchart values.yaml. You can remove it from the main values.yaml and add them only to the subcharts, so you don't set them twice.

Reply:
Currently all subchart configurations are configured in the values.yaml of this subchart. Only the configuration to enable a subchart is located in the main values.yaml. To be consistent this should also be the case for networkPolicy. You can remove it from the main chart values.yaml and add them in the related subchart values.yaml, so you don't configure it twice.

Reply:
Nice